### PR TITLE
Profile dropdown and other header style improvements

### PR DESCRIPTION
**Pull request by fabianmakila**
This PR includes the following:
- Dropdown for creating project/organization
- User profile menu dropdown
- Minor style adjustments to the header in general

### CODE REVIEW

File: frontend-new/src/components/layout/Header.vue [hunk 1]
```diff
@@ -123,27 +128,62 @@ authLog("render with user " + authStore.user?.name);
...
 
-      <div class="login-buttons flex gap-2 items-center">
-        <button class="flex mr-2" @click="theme.toggleDarkMode()">
+      <!-- Right side items -->
+      <div class="flex items-center gap-2">
+        <div v-if="authStore.user" class="flex items-center">
+          <DropdownButton name="Create">
+            <DropdownItem>Create project</DropdownItem>
+            <DropdownItem>Create organization</DropdownItem>
+          </DropdownButton>
```
Comment (NoahvdAa) [inline frontend-new/src/components/layout/Header.vue:140]: Should this be localised or do we not care about that yet for the new frontend?

Comment (MiniDigger) [reply frontend-new/src/components/layout/Header.vue:140]: they should, same with the dropdowns, there are even constants for them already iirc

**Review by MiniDigger — APPROVED** [review]
looks good else
I think ill do an i18n pass over the whole repo at some point, but would be cool to tackle them already now

**Comment by fabianmakila** [comment]
The dropdowns are now using i18n

**Comment by MiniDigger** [comment]
thanks!

**Pull request merged**